docs/en/about/faq.md

```diff
@@ -0,0 +1,40 @@
+# Frequently Asked Questions
+
+## What version of Python does Briefcase support?
+
+Python 3.10 or higher.
```

Comment:
For some reason this is rendering with a bigger gap than the other questions:

<img width="793" height="339" alt="Screenshot 2025-10-23 at 11 36 49" src="https://github.com/user-attachments/assets/80501911-183f-45f4-9941-b09b2e61be98" />